dev/com.ibm.ws.security.javaeesec.internal/src/io/openliberty/security/jakartasec/identitystore/permissions/IdentityStorePermissionServiceImpl.java

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.security.jakartasec.identitystore.permissions;
+
+import javax.security.enterprise.identitystore.IdentityStorePermission;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.security.javaeesec.identitystore.IdentityStorePermissionService;
+
+/**
+ * Implementation for Jakarta Security 1.0, 2.0 and 3.0 which all include
+ * the IdentityStorePermission class.
+ * 
+ * NOTE: For the Jakarta Security 2.0 and 3.0 implementations, they will 
+ * use the this bundle but transformed for jakarta instead of javax.
+ * 
+ * Jakarta Security 1.0 (appSecurity-3.0) = 
+ *     com.ibm.ws.security.javaeesec.internal.1.0_1.0.<VER>.jar
+
+ * Jakarta Security 2.0 (appSecurity-4.0) = 
+ *     io.openliberty.security.jakartasec.1.0.internal_1.0.<VER>.jar (transformed)
+
+ * Jakarta Security 3.0 (appSecurity-5.0)  = 
+ *     io.openliberty.security.jakartasec.1.0.internal_1.0.<VER>.jar (transformed)
+ *     
+ * *** appSecurity-3.0 was the first occurrence of the implementation (JSR375).
+ */
+
+@Component(
+        service = IdentityStorePermissionService.class,
+        immediate = true,
+        configurationPolicy = ConfigurationPolicy.IGNORE,
+        property = { "service.ranking:Integer=100"
+        })
+public class IdentityStorePermissionServiceImpl implements IdentityStorePermissionService {
```

Comment:
```suggestion
public class IdentityStorePermissionService {
```
Let's have this not depend on an interface and just have it be a normal class that can be referenced.

Reply:
Yes, have done as you've suggested.